DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/Centre/Configuration/RegistrationPromptsControllerTests.cs

```diff
@@ -325,11 +326,11 @@ public void AddRegistrationPromptConfigureAnswers_add_configures_new_answer_and_
             }
         }
 
-        [Test]
-        [TestCaseSource(
-            typeof(RegistrationPromptsControllerTests),
-            nameof(AddAnswerModelErrorTestData)
-        )]
+        //[Test]

```

Comment:
Why are these tests commented out?

Reply:
As observed the test was commented out because it depend on the GDS.MultiPageFormData concrete class which has been to the nugget parkage